public/app/percona/backup/components/BackupInventory/RestoreBackupModal/RestoreBackupModal.tsx

```diff
@@ -289,7 +289,6 @@ export const RestoreBackupModal: FC<RestoreBackupModalProps> = ({
                     name="serviceType"
                     label={Messages.serviceSelection}
                     fullWidth
-                    disabled={values.vendor !== DATABASE_LABELS[Databases.mysql]}
```

Comment:
Won't this allow all DB types? (not just mongo)

Reply:
You're right. I've updated the file.